arviz/plots/kdeplot.py

```diff
@@ -74,6 +75,8 @@ def plot_kde(
     contour : bool
         If True plot the 2D KDE using contours, otherwise plot a smooth 2D KDE.
         Defaults to True.
+    levels : list
+        Confidence levels for highest density interval contours of a 2D KDE.
```

Comment:
may be we can write  Confidence levels for highest density (2-dimensional) interval contours of a 2D KDE.

Reply:
```suggestion
        Confidence levels for highest density (2-dimensional) interval contours of a 2D KDE.
```
Like this? Saying 2D KDE at the end might be redundant, so I can change that to just KDE?

Reply:
This looks good, I'd keep the 2d kde as it's not too cluttery, but either alterntive is fine by me

Reply:
let's keep the 2d, a little redundancy is better in this case I think